packages/ui/src/components/Channel/EmptyChannelNotice.tsx

```diff
@@ -13,33 +17,42 @@ export function EmptyChannelNotice({
   channel: db.Channel;
   userId: string;
 }) {
-  const isGroupAdmin = useIsAdmin(channel.groupId ?? '', userId);
+  const { onPressGroupMeta } = useChatOptions() ?? {};
```

Comment:
When do you expect `useChatOptions` to not return anything? Only when the provider is uninitialized?

Trying locally on a freshly created multi-dm, this seems to no-op

Reply:
Ya, never, the hook should actually throw an error if that's the case -- good catch.